conf/machine/igx-orin-devkit.conf

```diff
@@ -0,0 +1,56 @@
+#@TYPE: Machine
+#@NAME: NVIDIA IGX Orin Devkit
+#@DESCRIPTION: NVIDIA IGX Orin Developer Kit
+
+NVPMODEL ?= "nvpmodel_p3701_0008_3740_0002"
+NVFANCONTROL ?= "nvfancontrol_p3740_0002_c01"
+
+require conf/machine/include/tegra234.inc
+require conf/machine/include/devkit-wifi.inc
+
+# Install the NVIDIA OOT modules.
+MACHINE_EXTRA_RRECOMMENDS:append = " nvidia-kernel-oot"
```

Comment:
It would be better to add specific modules here,  rather than the meta-package, which has many many drivers in it.  And several of those drivers don't look suitable for production use.

If the number of specific drivers is too large, we could adjust the packaging settings in the nvidia-kernel-oot recipe to group some of them together into a single package, then put that package here.

And please use an ordinary `+=` here, rather than `:append`.

Reply:
Thanks. I will check that and will update the recipe.